internal/pkg/deploy/cloudformation/stack/stack_local_integration_test.go

```diff
@@ -44,7 +48,15 @@ func Test_Autoscaling_Integration(t *testing.T) {
 	require.NoError(t, err)
 
 	t.Run("CloudFormation template must contain autoscaling resources", func(t *testing.T) {
-		path := filepath.Join("testdata", "autoscaling", wantedCFNTemplatePath)
+		path := filepath.Join("testdata", "stacklocal", wantedAutoScalingCFNTemplatePath)
+		wantedCFNBytes, err := ioutil.ReadFile(path)
+		require.NoError(t, err)
+		fmt.Println(tpl)
```

Comment:
just checking that this line is intentional

Reply:
🤦